xrdsst/controllers/service.py

```diff
@@ -66,6 +67,38 @@ def as_object(service):
         }
 
 
+class ServiceAccessListMapper:
+    @staticmethod
+    def headers():
+        return ['SS', 'CLIENT', 'DESCRIPTION', 'SERVICE', 'SERVICE_CLIENT', 'LOCAL_GROUP', 'NAME', 'RIGHTS_GIVEN', 'TYPE']
```

Comment:
Deleted local_group, but "ss", "client" and "description" should be kept because clients and descriptions with same values could exist on multiple security servers and it will be confusing when we would list duplicates without knowing which server or client these belong to